blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+- Webhook timeout for API server calls has been increased to 10 seconds. [#5175](https://github.com/knative/eventing/pull/5175)
+
+#### 🧹 Clean up
+
+- Do not set the finalizer for pingSource for pingSource controller, just deal with it in reconcilekind [#5002](https://github.com/knative/eventing/pull/5002)
+
+### Eventing Extensions
+
+
+#### Eventing Kafka Broker v0.22
+
+#### 🚨 Breaking or Notable
+
+- Kubernetes's minimum version is 1.18. [#779](https://github.com/knative-sandbox/eventing-kafka-broker/pull/779)
+
+#### 💫 New Features & Changes
+
+- Support ordered delivery.
+  - You can choose between ordered and unordered delivery through the label kafka.eventing.knative.dev/delivery.order in your triggers.
+  - Check out the docs for more details at https://knative.dev/docs/eventing/broker/kafka-broker . [#589](https://github.com/knative-sandbox/eventing-kafka-broker/pull/)
+
+#### 🐞 Bug Fixes
+
+- Add producer interceptor io.cloudevents.kafka.PartitionKeyExtensionInterceptor to provide ordered delivery based on the partitioning extension of the CloudEvents spec. [#751](https://github.com/knative-sandbox/eventing-kafka-broker/pull/751)
+- Fix unable to deploy KafkaSink without Kafka Broker installed [#714](https://github.com/knative-sandbox/eventing-kafka-broker/pull/714)
+
+### Client v0.22
+
+kn 0.22.0 comes with some bug fixes and minor feature enhancements. It's mostly a polishing release. If you are using the client API, there is a breaking change that was needed to align with Kubernetes client API's
+
+#### META
+The compile dependencies have been updated to Knative Serving 0.22.0 and Knative Eventing 0.22.0. There are no changes in the used API version for communicating with the backend.
+
+#### Managing custom domain mappings
+
+This release adds support for CRUD management of domain mappings. I.e. you can use `kn domain` along with its sub-commands `create`, `update`, `describe`, `list` and `delete` to fully manage DomainMapping resources for the installation of custom domains for Knative services.
+
+```
+# Create a domain mappings 'hello.example.com' for Knative service 'hello'
+kn domain create hello.example.com --ref hello
+
+# Update a domain mappings 'hello.example.com' for Knative service 'hello'
+kn domain create hello.example.com --refFlags hello
+
+# List all domain mappings
+kn domain list
+
+# Delete domain mappings 'hello.example.com'
+kn domain delete hello.example.com
+```
+See `kn domain help` for more information.
+
+
+#### Client API signature change
+
+A `context.Context` object is added as the first argument to every Client API method to align with Kubernetes' client API signatures. This change does not affect any client's CLI usage, only if the client-specific Golang API is used for communicating with the Knative backend has changed. The migration is straight forward, either pass an already existing context down to the call or leverage one of the available standard contexts (like `context.TODO()`).
```

Comment:
Be consistent with "Client API", it's "client API" earlier in the doc, please choose one and use throughout.

Reply:
Just to make you aware I didn't write this content. 
Is a copy and paste verbatim from the release notes, this blog post is just trying to put all existing release notes in a single place. The only change I do is some minor adjustments. 

https://github.com/knative/client/releases/tag/v0.22.0
![image](https://user-images.githubusercontent.com/1094878/115811618-06e95200-a3be-11eb-98b1-0dea36a797aa.png)

I think is @rhuss who writes the release notes for client

Reply:
I'm glad that tech writers are reviewing this notes, I will be happy to make any suggested changes, and then I can open an issue to the respective repos for someone with write access to release notes to update them there.